filebeat/etc/beat.yml

```diff
@@ -1,192 +1,189 @@
-################### Filebeat Configuration Example #########################
+###################### Filebeat Configuration Example #########################
```

Comment:
Perhaps we can remove the example part and use "Example" for the short one then.